app/models/restitution/cafe_de_la_place.rb

```diff
@@ -50,6 +50,10 @@ def initialize(campagne, evenements)
       super
     end
 
+    def complete?
+      evenements.any? { |e| e.nom == 'finSituation' }
```

Comment:
Attention aux string magique

```suggestion
      evenements.any? { |e| e.fin_situation? }
```